requirements.txt

```diff
@@ -1,8 +1,5 @@
-tsdat == 0.2.10
-netcdf4
-xarray
-numpy
-matplotlib
+tsdat >0.4.3, <=0.5.0
+cmocean  
+notebook
 act-atmos
-parse
-typer
```

Comment:
Typer was removed as a requirement, but it looks like it is still used in `runner.py`. I think this is probably OK as long as we continue to require it in `tsdat`, but I still would recommend adding it in here for good measure.